src/components/Contentful/DynamicRichText.vue

```diff
@@ -9,12 +9,16 @@ export default {
 		html: {
 			type: String,
 			default: ''
+		},
+		bodyColumns: {
+			type: Object,
+			default: () => {}
```

Comment:
I think the default might need to be `null`. Since `{}` evaluates as `true`, the grid classes will always be applied instead of the prose class.